app/user/rest.py

```diff
@@ -143,20 +147,20 @@ def send_user_sms_code(user_id):
     mobile = user_to_send_to.mobile_number if verify_code.get('to', None) is None else verify_code.get('to')
     sms_code_template_id = current_app.config['SMS_CODE_TEMPLATE_ID']
     sms_code_template = dao_get_template_by_id(sms_code_template_id)
-    verification_message = encryption.encrypt({
-        'template': sms_code_template_id,
-        'template_version': sms_code_template.version,
-        'to': mobile,
-        'personalisation': {
-            'verify_code': secret_code
-        }
-
-    })
-    send_sms.apply_async([current_app.config['NOTIFY_SERVICE_ID'],
-                          str(uuid.uuid4()),
-                          verification_message,
-                          datetime.utcnow().strftime(DATETIME_FORMAT)
-                          ], queue='notify')
+    notify_service_id = current_app.config['NOTIFY_SERVICE_ID']
+
+    saved_notification = persist_notification(template_id=sms_code_template_id,
+                                              template_version=sms_code_template.version,
```

Comment:
Nitpicky, but I don’t like this style of indentation. It means that if the name of the `persist_notification` function ever changes then you get a 6 line diff instead of a 1 line diff. I prefer:
```python
function(
    arg1,
    arg2,
)
```